homework6

najahizquierdo opened the pull request:
:| honestly, it runs on my computer but no output yet so ... lol

najahizquierdo committed:
homework6

:| honestly, it runs on my computer but no output yet so ... lol

najahizquierdo commented:
@dafrenchyman @brian004900

brian004900 commented on homework6/Dockerfile:
```diff
@@ -0,0 +1,17 @@
+
+COPY ./assignment6.sh assignment6.sh
+COPY ./baseballscripts.sql baseballscripts.sql
+
```
necessary file copied to the container

brian004900 commented on homework6/assignment6.sh:
```diff
@@ -0,0 +1,5 @@
+  echo "Baseball database DOES NOT exist "
+    mysql -h db -u root -e "CREATE DATABASE IF NOT EXISTS baseball"
+  else
+    echo "BASEBALL created"
```
This script worked to detect whether baseball exist in the database

brian004900 commented on homework6/baseballscripts.sql:
```diff
@@ -0,0 +1,54 @@
+  join game g
+  on b.game_id = g.game_id
+);
+
```
added index might increase speed.

brian004900 commented on homework6/baseballscripts.sql:
```diff
@@ -0,0 +1,54 @@
+CREATE TABLE historicalAverage(
```
I do not know how this sql is automatically executed, not included in the bash file.

brian004900 commented on homework6/docker-compose.yml:
```diff
@@ -0,0 +1,18 @@
+    image: baseball
+    volumes:
+      - ./results:/results
+    restart: unless-stopped
```
two containers created well.
volume linked to a local folder is a good idea to get the results.

najahizquierdo committed:
final commit for assignment 6 

Added updated files (in master) to pass all pre-commits! hopefully that is okay

najahizquierdo commented:
Final commit only uses 3 files in the main directory. I figured out why it wasnt running, the seperate directory caused issues :| Thanks!! Hope this all works well. ALSO, i updated the master with accurate files to pass the precommit. Let me know if you have any questions about anything! @dafrenchyman
